catalog/core/catalog-core-commands/pom.xml

```diff
@@ -199,22 +199,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.39</minimum>
+                                            <minimum>0.30</minimum>
```

Comment:
I thought the same thing but the added unit tests look pretty thorough at first glance.